File: src/main/page/typed-resources/activity.java
import android.app.Activity
import android.os.Bundle;
import android.widget.TextView;

public class MyActivity extends Activity
{
	@Override
	protected void onCreate( Bundle savedInstanceState )
	{
		super.onCreate( savedInstanceState );

		setContentView( R.layout.view );

		TextView title = (TextView) findViewById( R.id.my_title );
		title.setText( "Hello Java!" );
	}
}

File: src/main/page/typed-resources/activity.scala
class MyActivity extends Activity
{
	override def onCreate( savedInstanceState: Bundle ) =
	{
		super.onCreate( savedInstanceState )

		setContentView( R.layout.main )

		val title = findViewById( R.id.my_title ).asInstanceOf[TextView]
		title.setText( "Hello Scala!" )
	}
}